docs/on-prem.rst

```diff
@@ -37,6 +36,8 @@ This example has been validated with:
 Setup
 -----
 
+You can run this demo locally with Docker and follow the 3 steps below, or you can run it in a cloud IDE using `Gitpod.io <https://gitpod.io>`__ workspace by following this `link <https://gitpod.io/#https://github.com/confluentinc/cp-demo>`__.
```

Comment:
`pint` will work on cp-demo post PR merge.

> I need to test, but I was hoping that since 6.2.1-post is default, it will redirect to this branch

Does this mean Gitpod will always use whatever branch is default?  (Sorry, I don't know much about Gitpod).  Default branch in the repo right now is 6.2.1-post, does that mean a user who is looking at 6.2.0 version of docs, they click on this link expecting 6.2.0 version of cp-demo, but they get 6.2.1 version of cp-demo?  This will always be wrong/unexpected for older versions...

Reply:
I fairly sure gitpod uses the github link you put in `gitpod.io/#<github link>`, which would use the default branch.

To be explicit, you can specify the specific branch in the gitpod link, like I did earlier with https://gitpod.io/#https://github.com/vdesabou/cp-demo/blob/devx-2672 . RST might have a variable that can be used for the link? Might prefer to just use the default branch though.